Source/WebCore/Modules/WebGPU/GPUQueue.cpp

```diff
@@ -578,7 +578,7 @@ static void* copyToDestinationFormat(const uint8_t* rgbaBytes, GPUTextureFormat
             data[i0 + 1] = rgbaBytes[i + 1];
         }
 
-        sizeInBytes = sizeInBytes * sizeof(float);
```

Comment:
see malloc call on line 575, I thought about making a wrapper function but sizeInBytes is used in all the for loops